addons/base_import/static/src/import_action/import_action.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
     <t t-name="ImportAction" owl="1">
         <div class="h-100 d-flex flex-column">
-            <Layout className="'o_import_action d-flex h-100 overflow-auto bg-white'" display="display">
+            <Layout className="'o_import_action d-flex h-100 overflow-auto'" display="display">
```

Comment:
In dark mode, the table has a different background than the view one, this might be nitpicking but I'm wondering if we shouldn't replicate that in light mode

| Lightmode | Darkmode |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/128030743/c2bb302e-ac9e-4c12-b84f-1353809a2af3) | ![image](https://github.com/odoo/odoo/assets/128030743/15cf158e-24b5-40ef-8f71-913d53ab77f3) |